docs/src/modules/sdk/pages/views.adoc

```diff
@@ -111,7 +111,8 @@ include::example$key-value-customer-registry/src/main/java/customer/application/
 ----
 <1> Note we are adding a new handler to the existing table updater.
 <2> Marks the method as a delete handler.
-<3> An effect to delete the view row `effects().deleteRow()`. It could also be an update of a special column, to mark the view row as deleted.
+<3> Logical delete: use `effects().updateRow()` to keep the row but mark it as deleted by setting the `deleted` field to `true`.
+<4> Hard delete: use `effects().deleteRow()` to physically remove the row..
```

Comment:
At first I thought that we could deprecate `deleteRow` because auto-delete make it obsolete. But then I realise that once might decide if they want a soft or hard delete depending on some condition. 

Also, it should be possible to delete a row even it the entity itself it not deleted. For example, a logical deletion of the Entity with some custom event that triggers a hard delete (`deleteRow()`) call on the view side.